### PR TITLE
Create Delete Account Dialog Story

### PR DESCRIPTION
**Pull request by nkiruka**
This PR is to add stories for Delete Account Dialog.  This will be useful in viewing any changes that will be made to Delete Account Dialog component.

Screen shots
### Delete Student Account 
screen shot 2018-09-12 at 12 53 50 pm

### Delete Teacher Account
screen shot 2018-09-12 at 12 53 38 pm

### CODE REVIEW

File: apps/src/lib/ui/accounts/DeleteAccountDialog.story.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import DeleteAccountDialog from './DeleteAccountDialog';
+
+const PASSWORD = 'MY_PASSWORD';
+const DELETE_VERIFICATION = 'DELETE MY ACCOUNT';
+
+export default storybook => {
+  storybook
+    .storiesOf('DeleteAccountDialog', module)
```
Comment (Erin007) [inline apps/src/lib/ui/accounts/DeleteAccountDialog.story.jsx]: Please add to the Dialogs category ->   `.storiesOf('Dialogs/DeleteAccountDialog', module)`

Comment (nkiruka) [reply apps/src/lib/ui/accounts/DeleteAccountDialog.story.jsx]: Done!  Thanks.

Comment (nkiruka) [reply apps/src/lib/ui/accounts/DeleteAccountDialog.story.jsx]: Good points raised!  Thanks !

File: apps/src/lib/ui/accounts/DeleteAccountDialog.story.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+export default storybook => {
+  storybook
+    .storiesOf('Dialogs/DeleteAccountDialog', module)
+    .addStoryTable([
+      {
+        name: 'Delete Student Account',
+        description: 'Warning message for student account deletion',
+        story: () => (
+          <DeleteAccountDialog
+            isOpen={true}
```
Comment (maddiedierker) [inline apps/src/lib/ui/accounts/DeleteAccountDialog.story.jsx]: a lot of the props for this story and the next one are repeated. i would pull them out into a `DEFAULT_PROPS` object to make duplicating these stories easier in the future. there's an example [here](https://github.com/code-dot-org/code-dot-org/blob/4ba87b8b8553852f08f20954c701c45161e5ad71/apps/src/templates/manageStudents/MoveStudents.story.jsx#L35-L44)

Comment (nkiruka) [reply apps/src/lib/ui/accounts/DeleteAccountDialog.story.jsx]: Added default_props object.  Will create another PR 2 additional stories.

**Pull request merged**